mantidproject/mantid
# Re #21313 Remove Direct Beam

Removes the 'RegionOfDirectBeam' property from ReflectometryReductionOne and the Direct Beam input box from the Reflectometry GUI.

**To test:**
1. Ensure that the Direct Beam edit box is no longer visible in the Reflectometry GUI settings tab.
2. Ensure that the `RegionOfDirectBeam` property cannot be set when running `ReflectometryReductionOneAuto` from MantidPlot.
3. Ensure that the Reflectometry GUI continues to function as expected.
4. Code Review



Resolves #21313. 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
